example/discord_bot/basic_prefix.py

```diff
@@ -0,0 +1,34 @@
+import discord
+
+import vacefron
+
+from discord.ext import commands
+
+intents= discord.Intents.default()
+
+
+class Bot(commands.Bot):
+    def __init__(self, **kwargs):
+        super().__init__(command_prefix="!", intents=intents, **kwargs)
+        self.vac_api= vacefron.Client()
+        
+
+    #Optional
+    async def close(self):
+        await self.vac_api.close()
+        await super().close()
+
+bot= Bot()   
+intents.message_content= True
```

Comment:
Why is this here and not on line 7?
```suggestion
```

Reply:
> Why is this here and not on line 7?
> ```suggestion
> ```

Because intents.default does not enable message content intent tho, abd its required for making a prefixed command

Reply:
Yes but why is it 15 lines below the actual defining of intents

Reply:
i see